### ISSUE TITLE
Links to Anchors in a CMS Page Not Working

### ISSUE THREAD

**Issue by rcmp-randallw** [richmondrcmp/mobileapp#49]
In iOS a hyperlink to an anchor in the CMS page is not working:
![Link to Anchors](https://user-images.githubusercontent.com/47233113/70080414-58bbed00-15bb-11ea-86bd-311b614faa4d.png)

This functionality is working in Android.  

Note: I vaguely recall that perhaps this functionality does not work in iOS???

**Comment by jcarterOCV** [comment]
@rcmp-randallw You are correct in that this does not work on iOS with the way we present the HTML on the screen.
With that said, this is probably not something that we will be able to support at this time.

**Comment by rcmp-randallw** [comment]
Understood.
